src/python/pants/backend/python/lint/bandit/rules_integration_test.py

```diff
@@ -3,57 +3,50 @@
 
 from typing import List, Optional
 
-from pants.backend.python.lint.bandit.rules import BanditLinter
+from pants.backend.python.lint.bandit.rules import BanditConfiguration, BanditConfigurations
 from pants.backend.python.lint.bandit.rules import rules as bandit_rules
-from pants.backend.python.targets.python_library import PythonLibrary
+from pants.backend.python.rules.targets import PythonInterpreterCompatibility, PythonLibrary
 from pants.base.specs import FilesystemLiteralSpec, OriginSpec, SingleAddress
 from pants.build_graph.address import Address
-from pants.build_graph.build_file_aliases import BuildFileAliases
-from pants.engine.fs import FileContent, InputFilesContent, Snapshot
-from pants.engine.legacy.structs import PythonTargetAdaptor, PythonTargetAdaptorWithOrigin
+from pants.engine.fs import FileContent
 from pants.engine.rules import RootRule
 from pants.engine.selectors import Params
+from pants.engine.target import TargetWithOrigin
 from pants.rules.core.lint import LintResult
-from pants.source.wrapped_globs import EagerFilesetWithSpec
 from pants.testutil.option.util import create_options_bootstrapper
 from pants.testutil.test_base import TestBase
 
 
 class BanditIntegrationTest(TestBase):
 
-    good_source = FileContent(path="test/good.py", content=b"hashlib.sha256()\n")
+    good_source = FileContent(path="good.py", content=b"hashlib.sha256()\n")
```

Comment:
In most tests, I got rid of the `test` prefix to simplify things. In particular, with the `sources` field, we need to say just the filename, but everywhere else we would need the full path `test/good.py`. 

These formatters don't care at all about source roots, so this change made the tests simpler.